File: android-uitests/testData/MinSdk24App/app/src/main/java/android/com/app/MainActivity.java
package android.com.app;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        new Thread(() ->
        {
            Log.d("TAG", "Hello World from Lambda Expression");
        }).start();
    }
}

File: android/integration/testData/mindebugapp/src/main/java/com/example/mindebugapp/MainActivity.kt
package com.example.mindebugapp

import android.app.Activity
import android.os.Bundle
import android.util.Log
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this)
        label.setText("Hello Minimal Debug World!")
        Log.i("MainActivity", "Hello Minimal Debug World!")
        setContentView(label)
        Thread {
          while (true) {
            try {
              Thread.sleep(100)
            } catch (e: InterruptedException) {
              // do nothing
            }
          }
        }.start()
        Log.i("MainActivity", "Goodbye Minimal Debug World!")
    }
}
